examples/WPAShop/common/api.js

```diff
@@ -0,0 +1,27 @@
+export default {
+    async getProduct(id) {
+        const response = await fetch('https://fakestoreapi.com/products/' + id);
+        return await response.json();
+    },
+    async getProducts() {
+        const response = await fetch('https://fakestoreapi.com/products');
+        return await response.json();
+    },
+    async getAllCategories() {
+        const response = await fetch('https://fakestoreapi.com/products/categories');
+        return await response.json();
+    },
+    async getProductsByCategory(name) {
+        const response = await fetch('https://fakestoreapi.com/products/category/' + name);
+        return await response.json();
+    },
+    // async getProductsByPrice(from, to) {
```

Comment:
what is this code using for?

Reply:
This code is being used for query data from server